migrations/20170115_api_date_validation.js

```diff
@@ -0,0 +1,67 @@
+var migrationName = '20161230_nye_hats.js';
+var authorName = 'sprusr'; // in case script author needs to know when their ...
+var authorUuid = 'db711097-c5d0-4d25-8e5e-d955da4b79fd'; //... own data is done
+
+/*
+ * API date validation
+ */
+
+var mongo = require('mongoskin');
```

Comment:
the migration should be updated to use `monk` instead of mongoskin that doesn't support anymore our database version, see https://github.com/HabitRPG/habitica/blob/develop/migrations/20170120_missing_incentive.js#L10